common/changes/@microsoft/api-documenter/master_2019-03-26-19-58.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/api-documenter",
+      "comment": "Fix #1065: rendering of type unions in tables.",
```

Comment:
>Fix #1065: rendering of type unions in tables. [](start = 18, length = 46)

Can you update this to "Fix an issue with type unions rendering in markdown tables (addresses #1065)"

Reply:
Ok. I just pushed an update with this change.